app/settings.py

```diff
@@ -28,7 +28,12 @@
 FTP_HEARTBEAT_FOLDER = os.getenv('FTP_HEARTBEAT_FOLDER', '/heartbeat')
 
 RABBIT_QUEUE = os.getenv('RABBITMQ_QUEUE', 'sdx-survey-notifications')
+RABBIT_DELAY_QUEUE = os.getenv('RABBIT_DELAY_QUEUE', 'sdx-delayed-survey-notifications')
```

Comment:
The sdx bit was to namespace the queue to ensure there were no collisions with any potential EQ queues. This might be a non-issue now though with the two teams merging. The reason for the underscore rather than a hyphen was to try and convey it's relation to the main queue. 

How about `sdx-survey-notifications_retries`?